installer/targets/robot-user/setup

```diff
@@ -153,10 +75,15 @@ function github-cache-credentials() {
 
 # SET THE ROS IP IF ON ROBOT NETWORK
 
-# Set the ROS_IP if we are on the 192.168.2.* range and not on the real robot
+# Set the ROS_IP if we are on the 192.168.44.* range and not on the real robot
 if [ ! $ROBOT_REAL ] ; then
-	if `hostname -I | grep -q 192.168.2.` ; then
-		export ROS_IP=`hostname -I | sed 's/.*\(192\.168\.2\.[0-9]\+\).*/\1/'`
+	if `hostname -I | grep -q 192.168.44.` ; then
+		export ROS_IP=`hostname -I | sed 's/.*\(192\.168\.44\.[0-9]\+\).*/\1/'`
 	fi
 fi
 
+# DECLARE ALIAS FOR QUICK SSHf
```

Comment:
This will keep adding stuff to the file. First check if it is there already.

Or better, maybe auto enable it upon a tue-dev install?